modules/libSceHttp/entry.cpp

```diff
@@ -5,74 +5,439 @@
 #include "types.h"
 #include "utility/utility.h"
 
+#define WIN32_LEAN_AND_MEAN
+
+#include <Windows.h>
+#include <boost/asio.hpp>
+
+using namespace boost::asio;
+
 LOG_DEFINE_MODULE(libSceHttp);
 
 namespace {
 using SceHttpsCallback = SYSV_ABI int (*)(int libsslCtxId, unsigned int verifyErr, SceSslCert* const sslCert[], int certNum, void* userArg);
 
+#define ARRAY_LENGTH 128
```

Comment:
constexpr func = [](auto container, int id){} ? use std::array instead and call size()
or 
template<typename T, int size>
int testFunc(T(&)[size] container, int id){}